staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -27,6 +27,11 @@ const (
 	// More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions
 	labelPrefix = "batch.kubernetes.io/"
 
+	// CronJobScheduledTimestampAnnotation is a creation timestamp annotation for the Job.
+	// It records the original/expected creation timestamp for the running job.It is represented in RFC3339. Using
```

Comment:
this description doesn't seem accurate... it's not recording the creation timestamp, it's recording the scheduled timestamp

Reply:
Addressed.